numfocus-sdg-2023-r1.md

```diff
@@ -0,0 +1,133 @@
+The NumFOCUS small development grants program
+=============================================
+
+1 - Name of the submitter
+---------------------
+
+Jorge Martínez Garrido
+
+
+2 - E-mail direction
+--------------------
+
+contact@jorgemartinez.space
+
+
+3 - Nature of the software
+--------------------------
+
+Affiliated -> Other -> poliastro
+
+
+4 - Proposal title
+------------------
+
+Enhancing the visualization capabilities of poliastro
+
+
+5 - Two-sentence summary
+------------------------
+
+Implement a powerful and customizable visualization backend based on PyVista and
+provide plotting animation capabilities to poliastro.
+
+
+6 - Description of the proposal
+-------------------------------
+
+poliastro is an open source (MIT) collection of Python functions useful in
+Astrodynamics and Orbital Mechanics, focusing on interplanetary applications. It
+provides a simple and intuitive API and handles physical quantities with units.
+
+Visualizing astronomical objects together with their path and motion is a key
+feature of any Astrodynamics library, including poliastro.
+
+At the moment, poliastro supports two backends built on top of the Matplotlib
+library and the Plotly library. However, these libraries are not conceived for
```

Comment:
True. Let me rephrase this paragram so it becomes clear that althought the libraries support 3D visualization, their capabilities are limited.

Reply:
Regarding Cesium, I will make the following points:

* As of today, it only supports Earth as the attractor.
* It requires to use a browser or Jupyter to access the visualization.

On the other hand:

* Since PyVista is a generic graphics library, all sorts of attractors are supported at the cost of programming those by our means.
* PyVista runs in dedicated windows, notebooks or even in the web.